plugins/main/bridge/bridge.go

```diff
@@ -373,6 +440,16 @@ func setupVeth(netns ns.NetNS, br *netlink.Bridge, ifName string, mtu int, hairp
 		}
 	}
 
+	if vlans != nil {
+		// TODO setup vlan trunk
+		for _, v := range vlans {
+			err = netlink.BridgeVlanAdd(hostVeth, uint16(v), false, false, false, true)
+			if err != nil {
+				return nil, nil, fmt.Errorf("failed to setup vlan tag on interface %q: %v", hostIface.Name, err)
```

Comment:
may better to use `%w` rather than %v for err

Reply:
let me check the difference between %w and %v and feedback to you
tks for comment

Reply:
should we consider backward compatible to earlier golang?
if no, i will change to `%w`

Reply:
@tjjh89017 yes, please use %w. CNI follows current golang (1.17) and previous golang (1.16) which both have it. We are just a bit lazy in regularly bumping requirements.

Reply:
changed to `%w` 
thanks for suggestion